resolwe_bio/utils/test.py

```diff
@@ -84,14 +84,43 @@ def setUp(self):
         super(BioProcessTestCase, self).setUp()
         self.files_path = TEST_FILES_DIR
 
-    def prepare_genome(self):
-        """Prepare genome FASTA."""
-        inputs = {"src": "genome.fasta.gz",
-                  "bowtie_index": "bt_index.tar.gz",
-                  "bowtie2_index": "bt2_index.tar.gz",
-                  "bwa_index": "bwa_index.tar.gz",
-                  "hisat2_index": "hisat2_index.tar.gz",
-                  "subread_index": "subread_index.tar.gz"}
+    def prepare_genome(self, src='genome.fasta.gz'):
```

Comment:
Simplify. Use only `genome.fasta.gz` as the default genome. If the indices are specified, use them. Think about what to do with unspecified indices. Checks for existence of files are not needed.

Reply:
We decided that the function should not change because it was breaking a few things down the line. This function needs to either go full blown modular or remain as it is. Perhaps a TODO for future `self.`.